relay-event-schema/src/protocol/contexts/nel.rs

```diff
@@ -0,0 +1,54 @@
+#[cfg(feature = "jsonschema")]
+use relay_jsonschema_derive::JsonSchema;
+use relay_protocol::{Annotated, Empty, FromValue, IntoValue, Object, Value};
+
+use crate::processor::ProcessValue;
+
+/// Contains the unique to NEL reports information.
+///
+/// NEL is a browser feature that allows reporting of failed network requests from the client side.
+/// W3C Editor's Draft: <https://w3c.github.io/network-error-logging/>
+/// MDN: <https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging>
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct NelContext {
+    /// The age of the report since it got collected and before it got sent.
+    pub age: Annotated<i64>,
+    /// The type of the report.
+    #[metastructure(field = "type")]
+    pub ty: Annotated<String>,
```

Comment:
nit: If we make `type` an enum with an `other(String)` variant, we can probably shave off a string allocation for most use cases:

> The type of the network error may be one of the following pre-defined values from the specification, but browsers can add and send their own error types

https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging#dns.unreachable